tests/utils.py

```diff
@@ -85,3 +84,23 @@ def assert_histories_equal(actual_history, expected_history):
     assert _detach_timezone_stamp(actual_history) == _detach_timezone_stamp(
         (expected_history[0].astimezone(local_timezone), expected_history[1])
     )
+
+
+def assert_bookmarks_equal(actual_bookmarks, expected_bookmarks):
+    """Assert that two bookmarks are equal, accounting for differing timezones.
+
+    Use of any Python standard library timezone-management option, such as:
+
+        datetime.datetime.now(datetime.timezone.utc).astimezone().tzinfo
+
+    instead of a dedicated external library, notably here dateutil.tz, would
+    result in failures for certain timezones e.g. 'Europe/London' or
+    'Australia/Melbourne' due to DST effects.
+
+    Note that from Python >= 3.9 it is possible to use a built-in module,
+    zoneinfo (https://docs.python.org/3/library/zoneinfo.html) to achieve
+    timezone awareness. For earlier versions, an external module must be used.
+    """
+    assert _detach_timezone_stamp(actual_bookmarks) == _detach_timezone_stamp(
+        (expected_bookmarks[0], expected_bookmarks[1])
+    )
```

Comment:
This part won't work properly since bookmarks unlike history contains four fields. So to make it work
```python
assert _detach_timezone_stamp(actual_bookmarks) == _detach_timezone_stamp(
        (expected_bookmarks[0], *expected_bookmarks[1:])
    )
```

Reply:
Also change `_detach_timezone_stamp` implementation to
```python
return (hist[0].replace(tzinfo=None), *hist[1:])
```

Reply:
Also maybe change the paths to use `pathlib.Path` since they look different in say windows and they bookmarks won't actually be the same e.g. ("path\\to\\bookmark", "path/to/bookmark"). This will need the complete implementation to look something like
```suggestion
   actual = (*actual_bookmarks[:-1], Path(actual_bookmarks[-1]))
    expected = (
        expected_bookmarks[0].astimezone(tz.gettz()),
        *expected_bookmarks[1:-1],
        Path(expected_bookmarks[-1])
    )
    assert _detach_timezone_stamp(actual) == _detach_timezone_stamp(expected)
```

Reply:
> This part won't work properly since bookmarks unlike history contains four fields and also because you missed the timezone bit . So to make it work

Had done the time zone part before the force push ,was still failing for `chrome` and `chromium`

Good catch with the four fields